bindings/python/python.i

```diff
@@ -1,75 +1,265 @@
 %rename(__str__) reservedToString;
 %include "python/exception.i"
 
+%{
+#define SWIG_FILE_WITH_INIT
+%}
+
+%include "numpy.i"
+
+%init %{
+import_array();
+%}
+
 %{
 #include <iDynTree/Core/VectorDynSize.h>
+#include <iDynTree/Core/VectorFixSize.h>
```

Comment:
@traversaro not sure I need to include all these files. For example, I forgot to include `Wrench.h`, etc. and it seems to work.